Assets/Feature01/Scripts/Input/GamePadCursor.cs

```diff
@@ -0,0 +1,47 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using UnityEngine.InputSystem.LowLevel;
+using UnityEngine.InputSystem.Users;
+
+public class GamePadCursor : MonoBehaviour
```

Comment:
Gamepad cursor is not needed. Just use the direction vector of the Right Axis. Shoot at the direction.

Reply:
ji bhai baad ditesi eta